SROS2_Linux.md

```diff
@@ -120,24 +120,27 @@ To be able to use the ros2 CLI tools to interact with your secured system, you n
 export ROS_SECURITY_ENCLAVE_OVERRIDE=/talker_listener/listener
 ```
 
-Then use the CLI as usual:
+Then use the CLI with `--no-daemon` and `--spin-time`:
+
+> [!NOTE]
+> Avoid using `ros2 daemon` since it may not have security enclaves, and enough time duration should be given for the discovery in secured network.
 
 ```bash
-ros2 node list
+ros2 node list --no-daemon --spin-time 3
```

Comment:
```suggestion
ros2 node list --no-daemon --spin-time 4
```

Unclear how the value was picked at the time, but in the integration tests the value used is 4 so maybe we should use the samme here ?
https://github.com/ros2/sros2/blob/07608ab784bac6b295e7c031692f9bc6598adf76/sros2/test/sros2/commands/security/verbs/utilities/sros2_cli_test_case.py#L32
https://github.com/ros2/sros2/blob/07608ab784bac6b295e7c031692f9bc6598adf76/sros2/test/sros2/commands/security/verbs/utilities/sros2_cli_test_case.py#L72-L74

Reply:
sounds good.